src/rgw/rgw_common.h

```diff
@@ -1369,7 +1372,12 @@ struct req_state {
 
   req_state(CephContext* _cct, RGWEnv* e, RGWUserInfo* u);
   ~req_state();
+
+  void set_req_state_err(int err_no, RGWHandler* handler);
+  void set_req_state_err(int err_no, const string &err_msg, RGWHandler* handler);
```

Comment:
@mdw-at-linuxbox : Do you have any specific reason for keeping them within req_state, else I will move them out as free functions.